projects/pgai/pyproject.toml

```diff
@@ -70,29 +79,32 @@ target-version = "py310"
 docstring-code-format = true
```

Comment:
(This pertains to line 76, which is a few lines above this comment). Apparently we can drop the `target-version` from the ruff config if we have `project.requires-python` set (see https://docs.astral.sh/ruff/settings/#target-version)

Reply:
removed